src/main/resources/GoogleApps/Groovy.xml

```diff
@@ -604,16 +604,20 @@ public class GoogleAppsGroovy {
         return result;
     }
 
-    public retrieveFileFromGoogle(page, name, id, url) {
-        return retrieveFileFromGoogle(getDriveService(), page, name, id, url);
+    public retrieveFileFromGoogle(page, name, id, url, mediaType) {
+        return retrieveFileFromGoogle(getDriveService(), page, name, id, url, mediaType);
     }
 
-    public retrieveFileFromGoogle(driveService, page, name, id, url) {
+    public retrieveFileFromGoogle(driveService, page, name, id, url, mediaType) {
         addDebug("Retrieving ${name} to page ${page}: ${id} ${url}" )
         def adoc = xwiki.getDocument(page);
         try {
-            String mt = new Tika().detect(name);
```

Comment:
You should also remove the package import. Normally you should have also removed the Tika dependency, but this dependency wasn't declared.. (we were relying on transitive dependencies).